StatsdClientTests/StatsdClientTests.csproj

```diff
@@ -8,14 +8,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>StatsdClientTests</RootNamespace>
     <AssemblyName>StatsdClientTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Can we update this to run on .net core 2.0 (xplat) and ditch configuration?

Reply:
Sure. I will update the project and commit it along with the previous request.